src/build/index.js

```diff
@@ -56,11 +56,46 @@ const build = async (argv) => {
   return outfile
 }
 
+/**
+ * Build command
+ *
+ * @param {GlobalOptions & BuildOptions} argv
+ */
+const buildEsm = async (argv) => {
+  const dist = path.join(process.cwd(), 'dist')
+  // @ts-ignore no types
+  const ipjs = await import('ipjs')
+
+  await ipjs.default({
+    dist,
+    onConsole: (/** @type {any[]} */...args) => console.info.apply(console, args),
+    cwd: process.cwd(),
+    main: argv.esmMain,
+    tests: argv.esmTests
+  })
+}
+
 const tasks = new Listr([
   {
     title: 'Clean ./dist',
     task: async () => del(path.join(process.cwd(), 'dist'))
   },
+  {
+    title: 'Build ESM',
+    enabled: async ctx => {
+      const pkg = await fs.readJSON(path.join(process.cwd(), 'package.json'))
+
+      return pkg.type === 'module'
+    },
```

Comment:
```suggestion
    enabled: ctx => {
      const pkg = fs.readJsonSync(path.join(process.cwd(), 'package.json'))

      return pkg.type === 'module'
    },
```

`listr` enabled does not allow async functions